docs/java/how-to-emit-metrics-in-java.md

```diff
@@ -2,69 +2,32 @@
 id: how-to-emit-metrics-in-java
 title: How to emit metrics in Java
 sidebar_label: Emit metrics
+description: To emit metrics with the Java SDK, use the `MicrometerClientStatsReporter` implementation for Prometheus and other backends supported by Micrometer, and use `WorkflowServiceStubsOptions.Builder.setMetricsScope` to set the metrics scope in your Worker or Client code.
```

Comment:
Is it correct that we effectively copy-paste the first lines of the content into a description of the page? I thought there should be a short summary of what the page is all about, am I wrong?

Reply:
Yes, but in this case I've condensed this description further (see update).